lib/tapjoy/autoscaling_bootstrap/configure_autoscaler.rb

```diff
@@ -16,6 +20,11 @@ def initialize(misc_config:, aws_env:, user_data:, **unused_values)
 
       private
 
+      # Check if ASG exists so we know whether to create or update
+      def autoscaling_group_exists
+        !Tapjoy::AutoscalingBootstrap::AWS::Autoscaling::Group.describe.nil?
```

Comment:
Turns out activesupport is not included in this code flow, so I've reverted to using the previous code. Here are a couple of visual alternatives, which I don't find to be better in any compelling way:

```
Tapjoy::AutoscalingBootstrap::AWS::Autoscaling::Group.describe.nil? ? false : true
```

```
if Tapjoy::AutoscalingBootstrap::AWS::Autoscaling::Group.describe.nil?
  false
else
  true
end
# the unless form is no better
```